common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1366,35 +1368,44 @@ object CelebornConf extends Logging {
       .withAlternative("rss.metrics.system.enabled")
       .categories("master", "worker", "metrics")
       .doc("When true, enable metrics system.")
+      .version("0.2.0")
       .booleanConf
       .createWithDefault(true)
 
   val METRICS_SAMPLE_RATE: ConfigEntry[Double] =
     buildConf("celeborn.metrics.sample.rate")
       .withAlternative("rss.metrics.system.sample.rate")
       .categories("master", "worker", "metrics")
+      .doc("It controls if Celeborn collect timer metrics for some operations. Its value should be in (0.0, 1.0).")
+      .version("0.2.0")
       .doubleConf
-      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in [0.0, 1.0]")
+      .checkValue(v => v >= 0.0 && v <= 1.0, "should be in (0.0, 1.0)")
```

Comment:
"[" means inclusive "(" means exclusive, the code does not match the message

Reply:
Get it